data/css/modules/arrangement/_sideBySide.scss

```diff
@@ -1,27 +1,34 @@
 .ArrangementSideBySide {
     .CardTitle {
+        padding: 0px 8px;
+
         &__title {
             font-family: $title-font;
             color: $white-60-color;
-            padding: 13px 0px 3px 0px;
             font-size: 18px;
         }
 
         &:hover .CardTitle__title {
-            border-bottom: 1px solid $accent-light-color;
             color: $accent-light-color;
         }
 
         &.highlighted {
-            background-color: $primary-dark-color;
+            background-color: darken($background-dark-color, 5%);
 
             .CardTitle__title {
                 color: white;
             }
+        }
+    }
+    &__popover {
+        background-color: $primary-medium-color;
+        box-shadow: 2px 2px 6px black;
+        color: $white-60-color;
+        padding: .5em 0em;
+        min-width: 10em;
 
-            &:hover .CardTitle__title {
-                border-bottom: none;
-            }
+        .CardTitle {
+            padding: .4em 1em;
         }
     }
 }
```

Comment:
Looks like we need to update the hover color:

![virtualbox_endlessos_22_05_2017_10_53_33](https://cloud.githubusercontent.com/assets/481418/26315363/f25394e2-3ede-11e7-8ab7-5548aa9464ed.png)

![virtualbox_endlessos_22_05_2017_10_54_10](https://cloud.githubusercontent.com/assets/481418/26315385/fe6f64ea-3ede-11e7-809a-0904bf844bb1.png)

Otherwise is really hard to read